Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -584,6 +704,8 @@ protected theorem monotone (f : α →𝒄 β) : Monotone f :=
 theorem apply_mono {f g : α →𝒄 β} {x y : α} (h₁ : f ≤ g) (h₂ : x ≤ y) : f x ≤ g y :=
   OrderHom.apply_mono (show (f : α →o β) ≤ g from h₁) h₂
 
+set_option linter.deprecated false
+
 theorem ite_continuous' {p : Prop} [hp : Decidable p] (f g : α → β) (hf : Continuous' f)
```

Comment:
Why is this one not deprecated?

Reply:
Yes, good point

Reply:
Done, thanks.

Reply:
Sorry I think there was a misunderstanding, Crhistopher. The lemma should be deprecated, not deleted. And since the statement is stupid (we don't want to prove the lemma `P a → P b → P (ite p a b)` for every concrete proposition `P`) there is no need to write a version using `ωScottContinuous`.

Reply:
Ah, okay.